common/test/acceptance/pages/studio/users.py

```diff
@@ -256,6 +256,7 @@ def click_delete(self):
         self.q(css='.prompt .action-primary').click()
         self.wait_for_element_absence('.page-prompt .is-shown', 'Confirmation prompt is hidden')
         wait_for_ajax_or_reload(self.browser)
+        self.wait_for_page()
```

Comment:
Need to call wait_for_page on the CourseTeamPage object, not the UserWrapper object.